docs/operate/get-started/other-hardware/_index.md

```diff
@@ -929,7 +929,7 @@ The following attributes are available for `rdk:sensor:jessamy:weather:meteo_PM`
 
 {{% /tablestep %}}
 {{% tablestep number=2 %}}
-**Create a GitHub repo**
+**Create a GitHub repo and link to it from your `meta.json`**
```

Comment:
@JessamyT if there is more feedback you may want to consider separating this into two steps